src/Package/Impl/Commands/R/InstallRClientCommand.cs

```diff
@@ -20,7 +19,7 @@ internal sealed class InstallRClientCommand : PackageCommand {
         }
 
         protected override void Handle() {
-            var path = MicrosoftRClient.GetRClientPath();
+            var path = SqlRClientInstallation.GetRClientPath();
```

Comment:
No, I just wanted to shorten the name from MicrosoftSqlRClientInstallation :-) And yes, there is one that comes with SQL (maybe I should add 'SQL' to its name in the connections) and one installed separately.